server/controllers/userController.js

```diff
@@ -50,3 +51,52 @@ exports.updateUserInfo = async function (req, res) {
     return res.status(400).json({ massage: err });
   }
 };
+
+// Update name, avaterUrl and timeZone
+exports.signUpUser = async function (req, res) {
+  const name = req.body.name;
+  const email = req.body.email;
+  const avatarUrl = req.body.avatarUrl;
+  let userId = "";
+
+  // Check user has account
+  const existUser = await Users.find({ email: email }, { email: 1 });
+  console.log(existUser.length);
+  if (existUser.length > 0) {
+    return res.status(200).json({ massage: "Already Exist User Account" });
+  }
+
+  try {
+    // Create new User
+    const user = new Users({
+      name: name,
+      email: email,
+      avatarUrl: avatarUrl,
+      timeZone: "America/Toronto", // Default
+      calendarUrl: "",
+    });
+    userId = user._id;
```

Comment:
Can you explain what line 78 does. Cause i think user._id is not defined yet and might throw an error.